lib/index.js

```diff
@@ -8,6 +8,14 @@ const miniget = require('miniget');
 const m3u8stream = require('m3u8stream');
 const parseTime = require('m3u8stream/dist/parse-time');
 
+const current = 'v4.0.5';
+const message = '\x1B[31mWARNING:\x1B[0m ytdl-core is out of date! Update it using "npm i ytdl-core".';
+miniget('https://api.github.com/repos/fent/node-ytdl-core/releases/latest', { headers: { 'User-Agent': 'a/b' } })
+  .text().then(response => {
+    if (JSON.parse(response).tag_name !== current) {
+      console.error(message);
```

Comment:
i'd use `console.warn` since this is more of a warning. in node, it's the same as `console.error`, but in a browser it's printed differently

you can add `warn` to eslint's `noconsole` option https://eslint.org/docs/rules/no-console#options

Reply:
this has been done in the most recent commits